homeassistant/components/sensor/trafikverket_weatherstation.py

```diff
@@ -98,7 +104,7 @@ def update(self):
                     <FILTER>
                           <EQ name="Name" value='""" + self._station + """' />
                     </FILTER>
-                    <INCLUDE>Measurement.""" + air_vs_road + """.Temp</INCLUDE>
+                    <INCLUDE>Measurement.""" + air_vs_road + """.""" + keytype + """</INCLUDE>
```

Comment:
We cannot accept any PR for this platform until all protocol specific code has been extracted into a third party lib which is then used.